apps/extension/webpack.config.ts

```diff
@@ -0,0 +1,153 @@
+import * as dotenv from 'dotenv';
+import path from 'path';
+import CopyPlugin from 'copy-webpack-plugin';
+import HtmlWebpackPlugin from 'html-webpack-plugin';
+import url from 'url';
+// eslint-disable-next-line import/no-named-as-default
+import webpack from 'webpack';
+
+// Loads default vars from `.env.testnet` & `.env.mainnet` file in this directory.
+// Reference package.json build script. TODO: add `.env.mainnet` when ready.
+const envPath = process.env['NODE_ENV'] === 'mainnet' ? '.env.testnet' : '.env.testnet';
```

Comment:
typo here (twice .env.testnet)

Reply:
that was intentional 😅 , I'll make this clearer though

Reply:
a, I saw the TODO, but thought it was only related to creating an `.env.mainnet` file